src/decrypt.penumbra.worker.js

```diff
@@ -1,34 +1,60 @@
 /* eslint-disable class-methods-use-this */
 
+// external modules
+import 'regenerator-runtime/runtime';
 import * as Comlink from 'comlink';
-// import { decryptStream, getDecryptedContent } from 'src/index';
+import { fromWritablePort } from 'remote-web-streams';
+
+// local
+import fetchAndDecrypt from './fetchAndDecrypt';
+import onProgress from './utils/forwardProgress';
+import './transferHandlers/progress';
+
+// eslint-disable-next-line no-restricted-globals
```

Comment:
can we add `self` to a whitelist of allowed globals and remove all of the eslint-disables for this everywhere?

Reply:
I will look into seeing if there's a way to do this on a project-level basis.

Reply:
```js
"globals": {
  "self": true
}
```
https://eslint.org/docs/user-guide/configuring

Reply:
if you can't turn it off at that level, I would maybe remove the rule entirely. it's a pain to have to disable something that is used so frequently

Reply:
My code is what would go into `.eslintrc.json` at the top level (i.e. `"globals"` is at the same level as `"env"`

Reply:
The best thing would actually be just a top-of-file comment that specifies [`env`](https://eslint.org/docs/user-guide/configuring#specifying-environments)

So this file could have something that labels it as `serviceworker`

Reply:
/* eslint-env serviceworker */